src/OpenTracing/SpanOptions.php

```diff
@@ -33,6 +43,14 @@ public static function create(array $options)
         
         foreach ($options as $key => $value) {
             switch ($key) {
+                case 'is_active':
```

Comment:
The SpanOptions are for creating spans from the client POV. In that context is active or not is implicitly set by the call to either `startActiveSpan` or `startManualSPan`, so having this flag on options is redundant.